home/import_content_pages.py

```diff
@@ -268,7 +268,10 @@ def create_content_page_index_from_row(self, row: "ContentRow") -> None:
     def create_shadow_content_page_from_row(
         self, row: "ContentRow", row_num: int
     ) -> None:
-        locale = self.locale_from_display_name(row.locale)
+        if row.language_code:
+            locale = locale = Locale.objects.get(language_code=row.language_code)
```

Comment:
What if no Locale exists for the language code given in the row? In that case, we should raise an `ImportException`, similar to how `locale_from_display_name` does.